README.md

```diff
@@ -127,16 +185,23 @@ Run the web app container using:
 
     docker run --net='container:max-im2txt' -it webapp
 
-## Third-Party Sources
+# Sample Output
+
+![Web UI Screenshot](doc/source/images/webui.png)
+
+# Troubleshooting
+
+<!-- TODO: Add Common Troubleshooting Issues -->
+
+<!-- Idea: Explain cleanup page here? -->
+
+# Links
+
+<!-- TODO: Add Links -->
 
-This web app includes the following third-party javascript and css libraries
+# Learn More
 
-- [Bootstrap 3](https://getbootstrap.com)
-- [D3.js](https://d3js.org)
-- [d3-cloud](https://github.com/jasondavies/d3-cloud)
-- [Featherlight](https://noelboss.github.io/featherlight/)
-- [Glyphicons](http://glyphicons.com)
-- [Image Picker](http://rvera.github.io/image-picker/)
-- [JQuery](https://jquery.com)
```

Comment:
These links should go up in the "Links" section. For "Learn More", we have some pre-canned ones you can grab from any of the released code patterns.

Reply:
I moved these into the Featured Technologies section on advise from Steve (over Slack), do you think they should be in the links or learn more sections instead?

Reply:
In some cases, they go in both sections. But we can leave this for now.
The "Learn More" section should be IBM links to patterns and services. We can address this later.